sites/docs/docs/features/queries/sql-queries.md

```diff
@@ -11,17 +11,21 @@ When you open a page in dev mode, Evidence runs all of the queries on the page.
 You can include SQL queries in your page using a markdown code block (starting and ending with 3 backticks). Evidence requires a query name to be supplied directly after the first 3 backticks.
 
 ````markdown
-```data_sample
+```sales_by_country
 select country, sum(sales) as sales
 from international_transactions 
 ```
 ````
 
 ## Using Query Results
-All query results on a page are returned to a single object called `data`. To use a query result, you need to reference the query name as a subset of that `data` object. These references can be used in any of the components from our built-in library.
+All query results on a page are returned to a single object called `data`. To use a query result, you can to reference  a subset of the `data` object. For instance if your query is named `sales_by_country`, you can reference the results using `data.sales_by_country`.  For convinience, Evidence also exposes each query result by it's own name, e.g `sales_by_country`.  These references can be used in any of the components from our built-in library.
 
 For example, if your query name was `sales_by_country`:
 ```markdown
+<LineChart data={sales_by_country}/>
+```
+or alternatively,
+```markdown
 <LineChart data={data.sales_by_country}/>
 ```
 
```

Comment:
@udesh I think this is a bit confusing. 
I can't really imagine a user ever really using this {data.query} functionality, unless they were very advanced.

I'd replace line 21 with:
```
# Using Query Results 
Reference a query in a component using `data={query_name}`

For example, if your...
```

Reply:
Makes sense to me.  Fixed as suggested.